data/etc/apache2/conf-available/repro-plinth.conf

```diff
@@ -4,7 +4,5 @@
 ##
 <Location /repro>
     ProxyPass http://localhost:5080
-
-    Include includes/freedombox-auth-ldap.conf
-    Require ldap-group cn=admin,ou=groups,dc=thisbox
+    Include includes/freedombox-single-sign-on.conf
```

Comment:
@jvalleroy I think group restriction can be set if I made use of tokens in the auth-pubtkt ticket. I will try that.